packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -592,7 +605,7 @@ export const Table = <
       filterTypes,
       selectSubRows,
       data,
-      getSubRows,
+      getSubRows: (subComponent && getSubRowsWithSubComponents) ?? getSubRows,
```

Comment:
nit: Is this the same as a regular ternary?

```suggestion
      getSubRows: subComponent ? getSubRowsWithSubComponents : getSubRows,
```

Reply:
I have made it a regular ternary.